servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/TaskBasedAsyncSingleOperator.java

```diff
@@ -76,7 +77,22 @@ final io.servicetalk.concurrent.Executor executor() {
     @Override
     void handleSubscribe(final Subscriber<? super T> subscriber,
                          final CapturedContext capturedContext, final AsyncContextProvider contextProvider) {
-        original.delegateSubscribe(subscriber, capturedContext, contextProvider);
+        try {
+            original.delegateSubscribe(subscriber, capturedContext, contextProvider);
+        } catch (Throwable t) {
```

Comment:
For now I added this try-catch block for all calls to `super.handleSubscribe(...)` to satisfy tests. After thinking about it more, it feels like we should do that only for `executor().execute(...)` runnable, but not for cases that call this `super` method without offloading. Those cases likely should be caught by a subscriber. WDYT?